### PR TITLE
Add Alarm Clock

### PR DESCRIPTION
**Pull request by siddhantkhandelwal**
As part of the task given during python_sig

### CODE REVIEW

**Comment by Pro-Panda** [comment]
Thanks. Reviewed

Please do the following:
1. The commit message should not mention `Python SIG`. This will be irrelevant detail for future.
2. Use git CLI rather than copying file on GitHub. This helps to avoid copying errors
3. Add more features in the alarm. Add an option to ring daily/weekly etc
4. Update the readme as well

**Comment by siddhantkhandelwal** [comment]
Thanks for the input.
1. Will take care of commit messages
2. Had no idea of such copying errors. Thanks.
3. Will be adding features to the alarm clock soon.
4. Should I create a new readme file in the alarm clock directory, or update the original one?

**Comment by Pro-Panda** [comment]
> Should I create a new readme file in the alarm clock directory, or update the original one?

Update the original one

File: Readme.md [hunk 1]
```diff
@@ -27,7 +27,14 @@ This repository contains all of my Python scripts made for simplifying tasks.
...
+   - Plays the file 'alarm.mp3' in the cwd once the alarm expires.
+   - Will be adding other featues, like snooze, ring daily/weekly
+   - How to use:
+      ```bash
+      python alarm-clock-cli.py h m s
+      ```
 
 ### Improvement:
 Anyone is welcome to contribute and improve/modify the scripts. <br>
-Please open an issue if you find any bugs in any of the scripts.
```
Comment (Pro-Panda) [inline Readme.md]: Unnecessary change of newline.

File: Readme.md [hunk 2]
```diff
@@ -27,7 +27,14 @@ This repository contains all of my Python scripts made for simplifying tasks.
       ```bash
       python update.py /path/to/attendance_list /path/to/excel_sheet <Attd Column No.>
       ```
+3. Alarm-Clock:
```
Comment (Pro-Panda) [inline Readme.md]: Maybe change the entire stucture of the readme. Checkout this project's readme: https://github.com/Martinsos/edlib

**Comment by Pro-Panda** [comment]
Please see inline comments

**Comment by Pro-Panda** [comment]
Any progress ?

**Comment by siddhantkhandelwal** [comment]
The CLI client works perfectly on two test systems. The GUI client shall be completed at a later stage. The README file has all necessary instructions for using the alarm clock.

**Comment by Pro-Panda** [comment]
Thanks.

Reviewed, not tested
